pytube/__main__.py

```diff
@@ -123,8 +123,11 @@ def descramble(self) -> None:
         if 'streamingData' not in self.player_config_args['player_response']:
             config_response = get_ytplayer_config(
```

Comment:
lines 124-126 can be compressed to a single line, `config_response = get_ytplayer_config(self.watch_html)` now that the logic for setting `self.player_config_args` has been split off of it

Reply:
Indeed.